From 7d424c07fd0170cb183fd649213b184bfe55e248 Mon Sep 17 00:00:00 2001
From: Chen Lin Z <lin.z.chen@intel.com>
Date: Tue, 3 Sep 2019 13:03:33 +0800
Subject: [PATCH 1/2] [REVERT ME] disable pstore

pstore will block kernel startup on some boards.

Change-Id: I3080a60cd9bf205f37ece8b5e3fa2000469ae18a
Tracked-On:
Signed-off-by: Chen Lin Z <lin.z.chen@intel.com>
---
 aaas/mixins.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aaas/mixins.spec b/aaas/mixins.spec
index 33e6f409..caa396f4 100755
--- a/aaas/mixins.spec
+++ b/aaas/mixins.spec
@@ -20,7 +20,7 @@ dalvik-heap: tablet-10in-xhdpi-2048
 cpu-arch: slm
 allow-missing-dependencies: true
 dexpreopt: true
-pstore: ram_dummy(address=0x50000000,size=0x400000,record_size=0x4000,console_size=0x200000,ftrace_size=0x2000,dump_oops=1)
+pstore: false
 media: auto(add_sw_msdk=false, opensource_msdk=true)
 graphics: auto(gen9+=true,hwc2=true,vulkan=true,drmhwc=false,minigbm=true,gralloc1=true,enable_guc=false)
 storage: sdcard-mmc0-usb-sd(adoptablesd=true,adoptableusb=true)
-- 
2.21.0

